app/helpers/tests/GoogleData.spec.js

```diff
@@ -0,0 +1,47 @@
+import { makeTimelineObject } from './GoogleTakeoutUtils';
+const { location2, location3, location4 } = require('./fixtures/googleImport');
+
+const GOOGLE_IMPORT = {
+  timelineObjects: [
+    makeTimelineObject(location2),
+    makeTimelineObject(location3),
+    makeTimelineObject(location4),
+  ],
+};
+
+describe('mergeJSONWithLocalData', () => {
+  beforeEach(() => jest.resetModules());
```

Comment:
Seems like a code smell. is this needed?

Perhaps this is because of the module scoped jest.spyOn in other files?

Reply:
this is needed because each test is mocking the same dependency, probably better approach is to reset mocked module explicitly

Reply:
there is `jest.doMock('../../helpers/General'` in every test and mocks has to be reset in order to apply a new mock implementation.
BTW, from [.resetModules() docs](https://jestjs.io/docs/en/configuration#resetmodules-boolean):

> By default, each test file gets its own independent module registry. Enabling resetModules goes a step further and resets the module registry before running each individual test. This is useful to isolate modules for every test so that local module state doesn't conflict between tests. This can be done programmatically using jest.resetModules().

Which means .spyOn() called in other tests should not affect anything here

Reply:
ok great :)